docs/generated/packages/jest/documents/overview.md

```diff
@@ -22,14 +22,30 @@ First, install `@nx/jest`, if not already installed using your preferred package
 Make sure to install the `@nx/jest` version that matches the version of `nx` in your repository. If the version numbers get out of sync, you can encounter some difficult to debug errors. You can [fix Nx version mismatches with this recipe](/recipes/tips-n-tricks/keep-nx-versions-in-sync).
 {% /callout %}
 
+{% tabs %}
+{%tab label="npm"%}
+
 ```shell
-npm install --save-dev @nx/jest
+npm i --save-dev @nx/jest
 ```
 
+{% /tab %}
+{%tab label="yarn"%}
+
 ```shell
 yarn add --dev @nx/jest
 ```
 
+{% /tab %}
+{%tab label="pnpm"%}
+
+```shell
+pnpm install --dev @nx/jest
```

Comment:
why use the short hand of npm install but not pnpm?
we should probably be consistent 
this applies to a lot of the spots from what I can tell.

Reply:
I have completed these changes in this new PR #21226